quickwit/rest-api-tests/scenarii/es_compatibility/0024-delete_indices.yaml

```diff
@@ -0,0 +1,30 @@
+---
+method: POST
+api_root: http://localhost:7280/api/v1/
+endpoint: indexes/
+json:
+  version: "0.7"
+  index_id: test_index1
+  doc_mapping:
+    mode: dynamic
+sleep_after: 3
+---
+method: POST
+api_root: http://localhost:7280/api/v1/
+endpoint: indexes/
+json:
+  version: "0.7"
+  index_id: test_index2
+  doc_mapping:
+    mode: dynamic
+sleep_after: 3
+---
+method: DELETE
+api_root: http://localhost:7280/api/v1/
+endpoint: _elastic/test_index1,does_not_exist
+status_code: 404
```

Comment:
This should additionally support the query param `ignore_unavailable=true|false`  where specifying true ignore indexes that do not exist, and return a 200 if the delete operations can otherwise be carried out.

if for example, I'm trying to delete 100 indexes, and one of the doesn't exists, by default, this would fail the entire request. In most practical situations, it shouldn't matter if the index doesn't exist, We're trying to delete it anyway.

Reply:
I updated to handle `ignore_unavailable` and added a test for it